src/gsi/setuphowv.f90

```diff
@@ -324,6 +328,14 @@ subroutine setuphowv(obsLL,odiagLL,lunin,mype,bwork,awork,nele,nobs,is,conv_diag
         muse(i) = .true.
      endif
 
+!--  Note:
+!         The following if-block must NOT be applied for 2DRTMA (ie. twodvar_regional=.false.),
```

Comment:
@TingLei-NOAA 
Yes, for the operational (2D)RTMA, HOWV is a must. And it does not check the readyness of howv in firstguess. And i_howv_in_data is NOT applicable in 2DRTMA run.

For 3D analysis, HOWV would also be a necessary product of 3DRTMA. But since this code could be also used in other regional DA system, such as RRFS-DA. In those scenarios, HOWV is NOT a necessary field (like the U/V/T/Q/Ps) in firstguess.  So in 3D analysis, I would like to check the existence of HOWV and set i_howv_in_data to mark this.

Thanks,

Reply:
@GangZhao-NOAA  Thanks for the clarification.  If you are sure 2DRTMA means it has howv in first guess as required, why not just do like " if 2DRTMA ,   i_howv_in_data=1".  I feel even this is easier to understand than the current form . 
Of course, it is maybe better to add the evaluation of i_howv_in_data in the 2DRTMA too, which would also avoid possible mistake in 2DRTMA.

Reply:
My general comment may be missed. I would suggest we follow the same logic as all other observations.
i.e.: In setupxxx.f90, if required guess fields are NOT available in met bundle, stop GSI (which is done in the subroutine init_vars_).

Reply:
@guoqing-noaa  This is a good suggestion!

Reply:
@TingLei-NOAA 
This is good suggestion to check howv in firstguess use i_howv_in_data in 2DRTMA, but the firstguess used in (2D)RTMA is in binary format, it is not as easy as netcdf file to check the existence of a specific variable. Another reason is, less the modification to 2DRTMA code is better, since it is almost frozen as an operational system.
Thanks,